doc/manual/src/contributing/hacking.md

```diff
@@ -68,12 +68,20 @@ To build Nix itself in this shell:
 To install it in `$(pwd)/outputs` and test it:
 
 ```console
+[nix-shell]$ make check
 [nix-shell]$ make install
 [nix-shell]$ make installcheck -j $NIX_BUILD_CORES
 [nix-shell]$ ./outputs/out/bin/nix --version
 nix (Nix) 3.0
 ```
 
+To iterate on for example `libexpr`, prioritizing the most relevant checks:
+
+```console
+[nix-shell]$ J="-j $NIX_BUILD_CORES"
+[nix-shell]$ make $J libexpr-tests_RUN && make $J && make check && make install && make installcheck $J
```

Comment:
```suggestion
[nix-shell]$ make $J libexpr-tests_RUN && make $J && make check && make install && make tests/lang.sh.test installcheck $J
```

This may make lang.sh run sooner.

Reply:
There are indeed a few variations here and I wouldn't worry much about which is best :). We can always add more information later.